### ISSUE TITLE
Improve documentation and change DNS settings

### ISSUE THREAD

**Issue by zapodot** [constretto/constretto-core#16]
A new documentation site is under construction at http://constretto.github.io

When it is more complete, it should be used as main site for the constretto.org domain

**Comment by zapodot** [comment]
@kaarenilsen can you please resolve this issue? There is still people complaining about outdated documentation on constretto.org as well as missing input on the obsolete constretto JIRA..

A how-to for the dns change: https://help.github.com/articles/tips-for-configuring-an-a-record-with-your-dns-provider/

**Comment by trax72** [comment]
I like this framework, but how do I use things like `DataSourceConfiguration customerDataSource = configuration.at("datasources").from("customer").as(DataSourceConfiguration.class);` (taken from the wiki). 

There is no at() or from() methods... I guess they were removed at some point, but what's the alternative now?

**Comment by zapodot** [comment]
Hi @trax72! I am glad that you are considering using Constretto for your project. You are quite right, the "at" method was removed earlier (Constretto 2.0 I believe..) with no replacement. 

The sollution is to use the value field of the @Configuration annotation to refer to the configuration key yo want injected:

```java
public class DataSourceConfiguration {

   @Configuration("datasources.customer.jdbcUrl")
   private String jdbcUrl;

}
```
I have changed the example on the WIKI so it does not confuse anyone else..
